Update sync_thunderbird.rst (#12465)
nextcloud/documentation: user_manual/groupware/sync_thunderbird.rst

If you later want to add a new address book, you can redo all of those steps and only the books not already synchronized will be suggested.

> .. note:: If your account uses two-factor authentication, you need an application password for login rather than your
>    regular password. To configure an application password, log into the Nextcloud Web interface, click on the avatar
>    in the top right and choose *Personal settings*. Then choose *Security* in the left
>    sidebar and scroll to the very bottom. There you can create an app password (which can
>    also be revoked in the future without changing your main user password).

Reviewer: For maintainability probably better to only describe the creation process once:

```suggestion
.. note:: If your account uses two-factor authentication, you need a `dedicated app password <../session_management.html#managing-devices>`_ for login rather than your
   regular password.
```

If you later want to add a new address book, you can redo all of those steps and only the books not already synchronized will be suggested.

> .. note:: If your account uses two-factor authentication, you need a `dedicated app password <../session_management.html#managing-devices>`_ for login rather than your

Reviewer 2: Could you please link to the anchor https://github.com/nextcloud/documentation/blob/6bc5d2ce34d0813ed10810a01b97715949c9b029/user_manual/session_management.rst#L19? This will keep the link working even if the page/section moves. 

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-a-custom-anchor

If you later want to add a new address book, you can redo all of those steps and only the books not already synchronized will be suggested.

> .. note:: If your account uses two-factor authentication, you need a :ref:`dedicated app password <#managing_devices>` for login rather than your

Reviewer 2: I believe the syntax is incorrect with the #. See https://github.com/nextcloud/documentation/blob/585258b12d97924ff36689f2c46659319d26e7bc/user_manual/groupware/sync_osx.rst?plain=1#L29 as a reference.
